hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -10,6 +10,18 @@
 {
   name: "SENSOR_CTRL",
   clock_primary: "clk_i",
+  available_input_list: [
+    { name: "ast_debug_in",
+      desc: "ast debug inputs from pinmux",
+      width: "10"
```

Comment:
these are following the definitions inside `ast_pkg.sv` right now, which are out of date with the spec.
I'll wait for `ast_pkg.sv` to update in the next drop and change these accordingly.